api/pb/api.proto

```diff
@@ -12,7 +12,7 @@ message NewDBRequest {
 
 message CollectionConfig {
     string name = 1;
-    string schema = 2;
+    bytes schema = 2;
```

Comment:
Schema was always json data, so make it `bytes` like the rest of the json data we deal with